internal/helm/controller/reconcile.go

```diff
@@ -65,7 +65,7 @@ const (
 // release changes are necessary, Reconcile will create or patch the underlying
 // resources to match the expected release manifest.
 
-func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) { //nolint:gocyclo
```

Comment:
This will change with the new Helm pkg planed for the future so I am ok with ignoring the lint.